app/controllers/albums.js

```diff
@@ -0,0 +1,22 @@
+const snakeCaseKeys = require('snakecase-keys');
+const albumsService = require('../services/albums');
+
+const listAlbums = (_, res) => {
+  albumsService
+    .listAlbums()
+    .then(albums => res.send({ albums: albums.map(snakeCaseKeys) }))
+    .catch(error => error);
```

Comment:
You must send the error to errors middleware. The controller needs a third parameter to send things to the next middleware